### ISSUE TITLE
ZUIKI controller not recognized in Windows

### ISSUE THREAD

**Issue by ginga81** [leezer3/OpenBVE#902]
### Description

I follow the procedure, "Option" "Enable Densha de GO! contlloer plugin."
Pressed the settings button and download ZADIG.
'Press extract files (Windows)' link to download the cfg file.
But, there is no ZUIKI's .cfg file in this.
Only Type2.cfg, Shinkansen.cfg and Ryojouhen.cfg.
After installing Type2.cfg, ZUIKI appeared in the ZADIG list, so I selected it and installed it.
However, even if I restart OpenBVE, it is not recognized.
Perhaps the ZUIKI cfg file cannot be downloaded.

### Reproduction

This problem is not a route or vehicle problem, so I will omit it.

## Route

This problem is not a route or vehicle problem, so I will omit it.

## Train

This problem is not a route or vehicle problem, so I will omit it.

## Logs

I'll skip this issue as it doesn't crash.

### Related information 

* Operating system
Windows 10 Pro 64bit ver 22H2 OS Build 19045.3031
CPU:Intel (R) Core(TM) i5-4570 CPU @ 3.20GHz 3.20Ghz
RAM:16.0GB
Graphic board:NVIDIA GeForce RTX 3060
* Method of control (Keyboard, joystick, RailDriver)
Joystick:ZUIKI ZKNS-001

**Comment by ginga81** [comment]
When I connected it for the first time and started OpenBVE, it was not displayed in the list, so I decided that ZADIG was necessary and ran it.
At this point, it was already recognized as a general-purpose USB joystick, and I was playing with software that converts this joystick information from the keyboard to a general-purpose USB device in the JR East Simulator.
Since I installed the driver with ZADIG this time, not only did it not recognize it with a general-purpose USB joystick, but even if I deleted the device or deleted the driver (even with OpenBVE's customize), the driver installed with ZADIG was recognized like a zombie. And it is installed and there is nothing to do.
I'm a total novice when it comes to Windows, and it's only been a few days since I touched 10 for the first time in about 4 years since 7.
It's a very troublesome situation because I couldn't back to the general-purpose USB driver.

**Comment by ginga81** [comment]
I discovered a way to revert to the generic USB driver.
Press Windows Key + X and select Device Manager.
on Universal Serial Bus devices
Right click on One Handle MasCon for Nintendo Switch
Select update driver
Browse my computer for drivers
-> Select from the list of available drivers on your computer
Select the USB input device at the bottom and click "Next"
"Close" as it will be updated
I was able to revert back to the generic USB driver.

And I thought it wouldn't come out, so I put ZADIG in, but after returning to the general-purpose USB driver and looking at the list as you said
"HID-comploant game controller" was detected.
thank you!
I'm closing this as it's resolved.
Windows is difficult...
